### PR TITLE
#162601645: User should receive an  verification link after registration

### PR DESCRIPTION
**Pull request by emukungu**
#### What does this PR do?
Provide a user interface to enable the user reset his/her password

#### Description of Task to be completed?
Users should be able to presented with a reset password link

#### How should this be manually tested?
 - Clone the repository.
 - Install application dependencies using`yarn`
 - start the server using `yarn start:dev`
 - Enter `localhost:8080/signup` in the browser
 - Once successful, an email is sent to the user email account
 - Click the link in the email
 - The user will receive a successful verification message to continue and his redirected to login into his/her account

#### Any background context you want to provide?
The link in the backend should redirect the user to the user interface in the frontend

#### What are the relevant pivotal tracker stories?
#162601645

Screenshots
screen shot 2018-12-17 at 5 30 28 am

### CODE REVIEW

**Review by johnkegz — APPROVED** [review]
looks good

File: src/actions/userVerification.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  )
+   .then(response => response.json()
+    )
+    .then(value => {
+      dispatch({
+        type: "USER_VERIFIED",
+        payload: value
+      });
+      setTimeout(() => window.location.replace("/login"), 3000);
+      toastr.success("VERIFICATION COMPLETE. LOGIN");
```
Comment (shakirandagire) [inline src/actions/userVerification.js]: For consistency could you put your message in lowercase

File: src/components/AppRouter.jsx [hunk 2]
```diff
@@ -16,6 +16,9 @@ import ChangePasswordPage from "../containers/resetPassword/ChangePasswordContai
 import Profile from "../containers/Profile/Profile";
 import EditProfile from "../containers/Profile/EditProfile";
 import "../assets/css/styles.css";
+import UpdateUserinfo from "../containers/EmailVerification";
```
Comment (shakirandagire) [inline src/components/AppRouter.jsx]: Could rename the component to follow conventions

File: src/containers/EmailVerification.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import userVerified from "../actions/userVerification";
+import { connect } from "react-redux";
+
+
+export class UpdateUserinfo extends Component {
+  componentDidMount() {
+    const url = window.location.search;
+    const pk = url.replace("?token=", "");
```
Comment (shakirandagire) [inline src/containers/EmailVerification.jsx]: could you label the variable with relatable names

File: src/containers/EmailVerification.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import userVerified from "../actions/userVerification";
+import { connect } from "react-redux";
+
+
+export class UpdateUserinfo extends Component {
```
Comment (shakirandagire) [inline src/containers/EmailVerification.jsx]: could you rename the container name as requested above

**Review by shakirandagire — APPROVED** [review]
You have implemented all the feedback given during the sync.
Looks good to me

**Pull request merged**